oauth2/oauth2-core/src/main/java/org/springframework/security/oauth2/core/UserInfoRequestSchema.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2002-2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.security.oauth2.core;
+
+import java.io.Serializable;
+
+import org.springframework.security.core.SpringSecurityCoreVersion;
+import org.springframework.util.Assert;
+
+/**
+ * This section defines three methods of sending bearer access tokens in resource requests to resource servers.
+ *  Clients MUST NOT use more than one method to transmit the token in each request.
+ *
+ * <p>
+ * The OAuth 2.0 Authorization Framework defines three standard request schema types:
```

Comment:
I believe you meant `scheme` instead of `schema`? 

As per spec,

> 2.  **Authenticated** Requests

   This section defines three **methods**...

I believe the name `AuthenticationMethod` aligns best with the spec language.